frontend/ui/network/manager.lua

```diff
@@ -248,6 +248,9 @@ end
 function NetworkMgr:isConnected()
     if Device:isAndroid() or Device:isCervantes() or Device:isPocketBook() or Device:isEmulator() then
         return self:isWifiOn()
+    elseif Device:isKindle() then
+        local on, connected =  self:isWifiOn()
```

Comment:
I don't like changing the expected output from the function so conditionally. Although I suppose it's a completely backwards compatible expansion on the signature, provided this is made clear somewhere slightly more generic too.

Reply:
I thought this was cleaner than making the function just check for the file, and making a new function to detect if it is connected.